website/src/categories/components/combobox/code.md

```diff
@@ -0,0 +1,194 @@
+
+If you only specify a string array as the `options` property, the combobox will use the strings as both the label and value.
+
+</section>
+
+<section>
+
+## Grouping options
+
+You can group options using the `sl-option-group` element. This element can contain `sl-option` elements. It is not recommended to have nested `sl-option-group` elements.
+
+```html
+<sl-combobox>
+  <sl-listbox>
+    <sl-option-group label="Math">
+      <sl-option value="0">Algebra</sl-option>
+      <sl-option value="1">Geometry</sl-option>
+      <sl-option value="2">Calculus</sl-option>
+   </sl-option-group>
+    <sl-option-group label="Languages">
+      <sl-option value="3">English</sl-option>
+      <sl-option value="4">German</sl-option>
+      <sl-option value="5">French</sl-option>
+      <sl-option value="6">Spanish</sl-option>
+      <sl-option value="7">Chinese</sl-option>
+    </sl-option-group>
+  </sl-listbox>
+</sl-combobox>
+```
+
+If you are using the `options` property, you can group options by setting the `option-group-path` property. The component will automatically group options based on the value of the specified property. The value of the group property will be used to label the group.
+
+```js
+const options = [
+  { value: "0", label: "Algebra", group: "Math" },
+  { value: "1", label: "Geometry", group: "Math" },
+  { value: "2", label: "Calculus", group: "Math" },
+  { value: "3", label: "English", group: "Languages" },
+  { value: "4", label: "German", group: "Languages" },
+  { value: "5", label: "French", group: "Languages" },
+  { value: "6", label: "Spanish", group: "Languages" },
+  { value: "7", label: "Chinese", group: "Languages" }
+];
+```
+
+```html
+<sl-combobox
+  .options=${options}
+  option-group-path="group"
+  option-label-path="label"
+  option-value-path="value"
+></sl-combobox>
+```
+
+</section>
+
+<section>
+
+## Selected
+
+The combobox component is in single selection mode by default. If you want to enable multiple selection, you can set the `multiple` property on the `sl-combobox` element. The `value` property is, depending on the selection mode, either a single value or an array of values.
+
+There are two ways you can set the selected options for the combobox:
+1. By using the `value` property on the `sl-combobox` element
+2. By using the `selected` property on the `sl-option` elements
+
+It is sometimes easier to set the `selected` property since you are already rendering the options yourself. If you are not rendering the options yourself, you can specify the `group-selected-path` property instead.
+
+In both cases you could also use the `value` property on the `sl-combobox` element to set the selected options.
+
+Do not use both methods at the same time.
+
+</section>
+
+<ds-install-info link-in-navigation package="combobox"></ds-install-info>
```

Comment:
Maybe it would be better to move it (the installation info) to the line `59` to have it right after the example, just like we have it in other places?